tools/ci/cirrus_wheels.yml

```diff
@@ -23,10 +23,6 @@ linux_aarch64_task:
     # build in a matrix because building and testing all four wheels in a
     # single task takes longer than 60 mins (the default time limit for a
     # cirrus-ci task).
-    - env:
-        CIRRUS_CLONE_SUBMODULES: true
-        CIBW_BUILD: cp39-*
-        EXPECT_CPU_FEATURES: NEON NEON_FP16 NEON_VFPV4 ASIMD ASIMDHP ASIMDDP ASIMDFHM
     - env:
```

Comment:
Do we want to move the `EXPECT_CPU_FEATURES` line to the `3.10` test?

Reply:
It wasn't clear to me when I was modifying the ci where these are actually used.